data/sitesEN.json

```diff
@@ -5135,6 +5135,25 @@
     "tags": [
       "official"
     ]
+  },
+    {
+    "id": "en-scpsl",
+    "origins_label": "SCP: Secret Laboratory Fandom Wiki",
+    "origins": [
+      {
+        "origin": "SCP: Secret Laboratory Fandom Wiki",
+        "origin_base_url": "scp-secret-laboratory-official.fandom.com",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "SCP:_Secret_Laboratory_Official_Wiki"
+      }
+    ],
+    "destination": "SCP: Secret Laboratory Offical Wiki",
```

Comment:
Please remove "Official" from the title - it might be formally part of the title, but brevity is better since we don't have a lot of space in the UI